langgraph/src/pregel/write.ts

```diff
@@ -42,7 +43,7 @@ export class ChannelWrite<
         description: null,
         default: null,
         annotation: "TYPE_SEND",
-        isShared: true,
+        isShared: false,
```

Comment:
The default value is `false` in the Python implementation.

1. LangGraph: https://github.com/langchain-ai/langgraph/blob/main/langgraph/pregel/write.py#L55
2. LangChain: https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/runnables/utils.py#L419